src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,22 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 950px;
```

Comment:
looks like computed value (card-width and gap), isn't it?
lets calculate it here

Reply:
@Bohdan-Maksymiuk it's from a previous commit